src/main/auth.js

```
@@ -610,14 +617,33 @@ async function refreshAccessToken() {
+      fs.unlinkSync(TOKEN_FILE_PATH);
+      console.log('Local token file deleted successfully');
+    }
+
+    // Reset page group settings
+    const config = createConfigStore();
+    config.set('subscription', {
+      isAuthenticated: false,
+      isSubscribed: false,
+      expiresAt: '',
+      pageGroups: PAGE_GROUPS.ANONYMOUS
+    });
```

ESLint

[error] Missing trailing comma.

(comma-dangle)